galeracluster/source/documentation/galera-parameters.rst

```diff
@@ -2001,6 +2003,33 @@ The excerpt below is an example of how this Galera parameter might look in the c
    wsrep_provider_options="socket.checksum=2"
 
 
+.. _`socket.dynamic`:
+.. rst-class:: section-heading
+.. rubric:: ``socket.dynamic``
+
+.. index::
+   pair: wsrep Provider Options; socket.dynamic
+
+Enable connection engine to accept both SSL and TCP connections.   
+
+.. csv-table::
+   :class: doc-options
+
+   "Default Value", "false"
+   "Dynamic", "No"
+   "Initial Version", "4.8"
+
+By enabling this parameter, Galera should be possible to communicate with both SSL and TCP connections.

```

Comment:
By enabling this parameter, it should be possible for Galera to communicate with both SSL and TCP connections.

Reply:
Changed.